people.rkt

```diff
@@ -593,6 +540,12 @@
                         #:dissertation "https://www.williamjbowman.com/#dissertation"
                         #:extra "(University of British Columbia)"
                         #:personal-site "https://www.williamjbowman.com")
+              @(alumnus "Andrew Cobb" #:year 2019)
+              @(alumnus "Di Zhong" #:year 2019)
+              @(alumnus "Hyeyoung Shin" #:year 2019)
+              @(alumnus "Jonathan Schuster" #:year 2019
+                        #:extra "(Google)"
```

Comment:
`#:dissertation "http://hdl.handle.net/2047/D20318587"`

Reply:
Thanks! I couldn't find it :)